pkg/projects/resource_project.go

```diff
@@ -177,22 +198,38 @@ func projectResource() *schema.Resource {
 			})
 		}
 
+		if users != nil {
+			errors = packUsers(d, "user", users)
+		}
+
+		if errors != nil && len(errors) > 0 {
```

Comment:
I have come to believe that the `len` function can handle `nil` just fine; so it's probably safe to skip the `nil` check